doc/src/explanation/active-deprecations.md

```diff
@@ -76,7 +76,126 @@ SymPy deprecation warnings.
 
 ## Version 1.12
 
-There are no deprecations yet for 1.12.
+(deprecated-mechanics-joint-axis)=
+### New Joint intermediate frames
+
+The definition of the joint axis in the ``sympy.physics.mechanics`` has changed.
+Instead of using the arguments ``parent_axis`` and ``child_axis`` to
+automatically determine the joint axis and an intermediate reference frame. The
+joints now use an intermediate frame argument for both the parent and the child
+body, i.e. ``parent_interframe`` and ``child_interframe``. This means that you
+can now fully define the joint attachment, consisting of a point and frame, for
+both bodies. Furthermore, if a joint like the ``PinJoint`` has a specific joint
+axis, e.g. the axis around which the rotation occurs, then this axis can be
+specified using the ``joint_axis`` argument. An advantage of this setup is that
+one can more accurately define the transformation from the parent body to the
+child body.
```

Comment:
```suggestion
The definition of the joint axis in the ``sympy.physics.mechanics`` has changed.
Instead of using the arguments ``parent_axis`` and ``child_axis`` to
automatically determine the joint axis and an intermediate reference frame, the
joints now use an intermediate frame argument for both the parent and the child
body, i.e. ``parent_interframe`` and ``child_interframe``. This means that you
can now fully define the joint attachment, consisting of a point and frame, for
both bodies. Furthermore, if a joint like the ``PinJoint`` has a specific joint
axis, e.g. the axis about which the rotation occurs, then this axis can be
specified using the ``joint_axis`` argument. An advantage of this setup is that
one can more accurately define the transformation from the parent body to the
child body.
```

Reply:
Changed all "around"s to "about"s for consistency ([both are allowed](https://english.stackexchange.com/questions/45566/rotate-about-vs-rotate-around))